be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.h

```diff
@@ -10,8 +10,8 @@
 namespace starrocks::pipeline {
 class AggregateBlockingSinkOperator : public Operator {
 public:
-    AggregateBlockingSinkOperator(int32_t id, int32_t plan_node_id, AggregatorPtr aggregator)
-            : Operator(id, "aggregate_blocking_sink", plan_node_id), _aggregator(std::move(aggregator)) {
+    AggregateBlockingSinkOperator(OperatorFactory* factory, int32_t id, int32_t plan_node_id, AggregatorPtr aggregator)
```

Comment:
Why pass OperatorFactory to operator?

Reply:
both runtime in-filters, bloom-filters are kept in OperatorFactory, shared by multiple Operator instance. derived Operator access runtime filters from its OperatorFactory.

Reply:
I don't thinks this is reasonable.  you could directly pass bloom-filters to some operator, in filters only need to pass scan operator

Reply:
It's just two different implementations,  the modification involving interface adjustments are very tedious, this problem can be tackled in another PR.